packages/bpk-docs/src/pages/ButtonsPage/ButtonsPage.js

```diff
@@ -51,7 +51,7 @@ const AlignedBpkLargeHelpIcon = alignToLargeButton(TestBpkLargeHelpIcon);
 const AlignedBpkSmallSearchIcon = alignToButton(TestBpkSmallSearchIcon);
 const AlignedBpkLargeSearchIcon = alignToLargeButton(TestBpkLargeSearchIcon);
 
-class LoadingButtonContainer extends React.Component<{}, { loading: boolean }> {
+class LoadingButtonContainer extends Component<{}, { loading: boolean }> {
   constructor() {
     super();
 
```

Comment:
From what I can see, you've not changed anything in this file to make it work with the `import React, { Component } from 'react';` syntax. Doesn't Flow complain?